holmes/plugins/toolsets/azuremonitor_metrics/utils.py

```
@@ -0,0 +1,714 @@
+"""Utility functions for Azure Monitor Metrics toolset."""
+
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)